src/constants/howConst.js

```diff
@@ -6,4 +14,183 @@ export const HowData = {
   step4: `04.\nWash containers\nat commercial facility`,
   highlighter: 'Circular ecosystem',
   buttonText: 'Work with Us'
+};
+
+export const HowInfo = {
```

Comment:
This is awesome.  Great work refactoring this, it looks like it was a huge pain, but it turned out great.